wol 1.0: bad checksum

CC @patricoferris @RyanGibb 
I checked manually: error persists unless it is a MiM attack.

```
[ERROR] Got some errors while processing: wol.1.0, wol-eio.1.0, wol-mirage.1.0
  - [wol.1.0] https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz (Bad
    checksum, expected
    sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4)
  - [wol-eio.1.0] https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz (Bad
    checksum, expected
    sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4)
  - [wol-mirage.1.0] https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz
    (Bad checksum, expected
    sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4)
```

--- a/packages/wol-eio/wol-eio.1.0/opam
+++ b/packages/wol-eio/wol-eio.1.0/opam
@@ -36,7 +36,7 @@ build: [
 dev-repo: "git+https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan"
 url {
   src:
-    "https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz"
+    "https://github.com/ocaml/opam-source-archives/raw/main/wol-1.0.tar.gz"
   checksum: [
     "md5=e7a5baecc01a778d3772ad52a5b597f0"
     "sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4"

--- a/packages/wol-mirage/wol-mirage.1.0/opam
+++ b/packages/wol-mirage/wol-mirage.1.0/opam
@@ -33,7 +33,7 @@ build: [
 dev-repo: "git+https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan"
 url {
   src:
-    "https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz"
+    "https://github.com/ocaml/opam-source-archives/raw/main/wol-1.0.tar.gz"
   checksum: [
     "md5=e7a5baecc01a778d3772ad52a5b597f0"
     "sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4"

--- a/packages/wol/wol.1.0/opam
+++ b/packages/wol/wol.1.0/opam
@@ -34,7 +34,7 @@ build: [
 dev-repo: "git+https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan"
 url {
   src:
-    "https://tangled.org/ryan.freumh.org/ocaml-wake-on-lan/archive/1.0.tar.gz"
+    "https://github.com/ocaml/opam-source-archives/raw/main/wol-1.0.tar.gz"
   checksum: [
     "md5=e7a5baecc01a778d3772ad52a5b597f0"
     "sha512=a254cefa47360b7e60c875db254062645c629a95f063cd500955109e18ebd2918f1d977024747f17841ad02e8f2d911deb158552e85459b23d0c5772d52e5bb4"

